src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIPacket.NetStandard.cs

```diff
@@ -18,31 +19,39 @@ internal partial class SNIPacket
         /// <param name="callback">Completion callback</param>
         public void ReadFromStreamAsync(Stream stream, SNIAsyncCallback callback)
         {
-            // Treat local function as a static and pass all params otherwise as async will allocate
```

Comment:
Can you retain the comment?

Reply:
I can but it's no longer needed since a static can't refer to outer locals, sure you want to keep it?